README.md

```diff
@@ -1,6 +1,6 @@
-# Template D-Bus service
+# Connectivity Manager
 
-Template for creating a D-Bus service component.
+D-Bus service for managing connectivity. TODO: Write more here.
```

Comment:
Please specify how to build and what dependencies there are. Does not need to be overly complicated but at least state something like "build using regular Meson setup". I do not know enough about meson to say if such a thing exists anyways... Also state how to run the tests.

Reply:
Added basic build instructions in template and rebased this pr to that template revision (since it is the "first commit", it is fine to do this).